eng/cibuild_bootstrapped_msbuild.ps1

```diff
@@ -64,7 +64,15 @@ try {
 
   if ($buildStage1)
   {
-    & $PSScriptRoot\Common\Build.ps1 -restore -build -ci -msbuildEngine $msbuildEngine /p:CreateBootstrap=true @properties
+    # Run tests on Full Framework instead of the commented code below. 
```

Comment:
```suggestion
    # Run tests on the first-stage build instead of the normal bootstrapped build since the VS environment in hosted machines is too new to work for this old branch. 
```